src/plugins/console/public/lib/utils/index.ts

```diff
@@ -117,24 +117,23 @@ export const replaceVariables = (
   requests: RequestArgs['requests'],
   variables: DevToolsVariable[]
 ) => {
-  const urlRegex = /(\${\w+})/g;
-  const bodyRegex = /("\${\w+}")/g;
+
+  const urlRegex = /\${(\w+)}/g;
+  const bodyRegexSingleQuote = /(?<!")"\${(\w+)}"(?!")/g;
+  const bodyRegexTripleQuotes = /"""\${(\w+)}"""/g;
+
   return requests.map((req) => {
     if (urlRegex.test(req.url)) {
-      req.url = req.url.replaceAll(urlRegex, (match) => {
-        // Sanitize variable name
```

Comment:
I omitted these sanitizing lines by utilizing a capture group in the regex because this chunk makes my adding code verbose and this function too long.

- The old regex `(\${\w+})` includes not only `\w+` but also `\${` and `}` in the the parentheses.
- The new regex `\${(\w+)}` includes only `\w+` in the parentheses, which is referrable later as `key`.